controllers/front/DiscountController.php

```diff
@@ -39,15 +39,26 @@ public function initContent()
     {
         parent::initContent();
 
-        $cart_rules = CartRule::getCustomerCartRules($this->context->language->id, $this->context->customer->id, true, false);
+        $cart_rules = CartRule::getCustomerCartRules($this->context->language->id, $this->context->customer->id, true, true, true);
```

Comment:
@yanngarras This issue was fixed in #5978, which is included [in v1.6.1.7](http://build.prestashop.com/news/prestashop-1617-maintenance-release/). Please upgrade any 1.6.1.6 shop to 1.6.1.7!